### DISCUSSION TITLE
Remove text area from Snackbar if message string is null/empty

### DISCUSSION THREAD

**Discussion by ZitaRR** [CommunityToolkit/Maui#960] [New Feature Discussions]
Currently the ``Snackbar.Make`` method requires a message string to be passed in. I'm in need of an option to not pass in the message because I want the message to be rendered within the action button. Obviously this is possible by just sending in an empty string as the message and supply the real message to the ``actionButtonText`` parameter. However, this creates a small empty space on the left side inside of the Snackbar which is quite an eyesore. I've taken a quick look in the Toast/Snackbar classes, perhaps a possible solution would be to remove the padding from the component containing the message but I'm not entirely sure. 

Another option could be to add an OnClick event to Toasts and just use that instead.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
